autoscheduler/frontend/src/tests/redux/CourseCards.test.ts

```diff
@@ -274,4 +284,94 @@ describe('Course Cards Redux', () => {
       });
     });
   });
+
+  describe('clearCourseCards', () => {
+    test('deletes all course cards and resets numCardsCreated', () => {
+      // arrange
```

Comment:
I mean, there was no data to clear in this case, so this test would've passed even without dispatching any actions. I'd probably add a card during the arrange phase just to make sure that clearing it actually does something.